content/en/blog/2023/ambient-ebpf-redirection/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Leverage eBPF for traffic redirection in Istio ambient mode"
+description: An alternative approach to redirect application pod traffic to ztunnel in Istio ambient mode.
+publishdate: 2023-03-15
+attribution: "Iris Ding (Intel), Chun Li (Intel)"
+keywords: [istio,ambient,ztunnel,eBPF]
+---
+
+The istio-cni component running on each Kubernetes worker node is responsible for redirecting application pod traffics to ztunnel on that node. By default it relies on iptables and
+[Geneve](https://www.rfc-editor.org/rfc/rfc8926.html) tunnel to achieve this redirection.  Now a new approach which is based on eBPF is also available in Istio ambient mode for this purpose.
+
+## Why eBPF
+
+In the context of Istio ambient mode redirection, although performance is undoubtedly important, it is imperative to recognize that, programmability or possibility takes precedence. With eBPF, you can leverage additional context to make these changes in the kernel so that packets can bypass complex routing and simply arrive at their final destination, that is sought-after for redirection datapath. Moreover, eBPF will provide the same or better control in datapath compared with iptables while providing the end of user experience that we are looking for.

```

Comment:
> programmability or possibility takes precedence

What do you mean by `possibility takes precedence`? Possibility of what?

> eBPF will provide the same or better control

Better how? Are there limitations with iptables, that only eBPF can overcome?

> providing the end of user experience that we are looking for.

What end user experience are you referring to? The redirection?